Packs/SplunkPy/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Splunk",
     "description": "Run queries on Splunk servers.",
     "support": "xsoar",
-    "currentVersion": "3.2.17",
+    "currentVersion": "3.3.0",
```

its a small change that requires a revision release version not a minor one, please update the name of the release note as well.
```suggestion
    "currentVersion": "3.2.18",
```